var/spack/repos/builtin/packages/pcre2/package.py

```diff
@@ -28,3 +28,10 @@ def configure_args(self):
             args.append('--enable-pcre2-32')
 
         return args
+
+    @property
+    def libs(self):
```

Comment:
Isn't this basically the default method?

Reply:
Ah, I see, you had to add the `*` because pcre2 has a weird 8, 16, 32 extension on its library names.